docs/reference/commandline/secret_ls.md

```diff
@@ -20,9 +20,6 @@ Usage:	docker secret ls [OPTIONS]
 
 List secrets
 
-Aliases:
-  ls, list
```

Comment:
hm, instead of removing here, we probably should _add_ the alias (because `docker images ls` and `docker volumes ls` do have this as an alias)